examples/controlpanel/package.json

```diff
@@ -27,6 +27,7 @@
     "prepublishOnly": "npm run export"
   },
   "dependencies": {
+    "lodash": "^4.17.15",
```

Comment:
As publish the npm package as pre-bundled, all dependencies have to go into "devDependencies"